specification/appconfiguration/AppConfiguration/routes.tsp

```diff
@@ -9,6 +9,13 @@ using Azure.Core.Traits;
 
 namespace AzureAppConfiguration;
 
+alias Body<T, TName extends valueof string = "body", TDoc extends valueof string = "The body parameter."> = {
```

Comment:
There is also already bodyparameter type in azure core

Reply:
I saw the Body<> template in typespec/http and tested it out in the playground